tests/Aspire.Hosting.Tests/Redis/AddRedisTests.cs

```diff
@@ -69,7 +69,24 @@ public void AddRedisContainerAddsAnnotationMetadata()
     }
 
     [Fact]
-    public async Task RedisCreatesConnectionString()
+    public void RedisCreatesConnectionString()
+    {
+        var appBuilder = DistributedApplication.CreateBuilder();
+        var password = "p@ssw0rd1";
+        appBuilder.Configuration["Parameters:pass"] = password;
+
+        var pass = appBuilder.AddParameter("pass");
+        appBuilder.AddRedis("myRedis", password: pass).PublishAsContainer();
+
+        using var app = appBuilder.Build();
+
+        var appModel = app.Services.GetRequiredService<DistributedApplicationModel>();
+
+        var connectionStringResource = Assert.Single(appModel.Resources.OfType<IResourceWithConnectionString>());
+        Assert.Equal("{myRedis.bindings.tcp.host}:{myRedis.bindings.tcp.port},password={pass.value}", connectionStringResource.ConnectionStringExpression.ValueExpression);
+    }
+    [Fact]
+    public async Task RedisCreatesConnectionStringWithPassword()
```

Comment:
Should this be `*WithoutPassword`?
```suggestion
    public async Task RedisCreatesConnectionStringWithoutPassword()
```